libqtile/extension/dmenu.py

```diff
@@ -89,11 +89,8 @@ def _configure(self, qtile):
             self.configured_command.extend(("-h", str(self.dmenu_height)))
 
     def run(self, items=None):
-        if items:
-            if self.dmenu_lines:
-                lines = min(len(items), self.dmenu_lines)
-            else:
-                lines = len(items)
```

Comment:
Why this change? Won't it break if `self.dmenu_lines` is `None`, as it is by default?

Reply:
`if None` will return `False`, so it should not broke anything.

Reply:
Oh yeah, right!